src/template.rs

```diff
@@ -23,17 +23,28 @@ fn template_path(git: Option<&str>) -> TemplatePath {
     // Use a minimal template by default.
     const DEFAULT_REPOSITORY: &str = "https://github.com/TheBevyFlock/bevy_new.git";
     const DEFAULT_BRANCH: &str = "main";
+    const QUICKSTART_REPOSITORY: &str = "https://github.com/TheBevyFlock/bevy_quickstart.git";
```

Comment:
Could you rename these consts to `NEW_2D_REPOSITORY`? I'd like to treat `quickstart` as just the working title until it gets merged into Bevy. The `NEW_` prefix is due to identifiers not being allowed to start with a number.